### DISCUSSION TITLE
No access to GitHub Copilot found

### DISCUSSION THREAD

**Discussion by sjackman** [community/community#19213] [Copilot Conversations]
I followed the instructions at https://github.com/github/copilot-docs/blob/main/docs/visualstudiocode/gettingstarted.md#installing
I created a file `copilot.js` and open it in VSCode. I get the error message `No access to GitHub Copilot found`. I click the button `Signup for GitHub Copilot`, and under `GitHub Copilot` it says `You are eligible to use GitHub Copilot for free`. How do fix this error message `No access to GitHub Copilot found`?

<img width="460" alt="image" src="https://user-images.githubusercontent.com/291551/175135761-093378b0-e6c4-4302-9d2b-9c4c223b1046.png">
<img width="446" alt="image" src="https://user-images.githubusercontent.com/291551/175135926-75fc9735-b718-41d6-9e87-0b9b18c8639b.png">

**Comment by melvincwng** [comment]
@sjackman, try the following:

1. Log out of your github account in VSCode
2. Close your VSCode editor
3. Relog in back to your Github account using VSCode
4. This should regenerate the auth token from Github's servers
5. If you have access to Github Copilot, it should be working now.

These are the steps which worked for me... so give it a try and hopefully they work for you too.

**Reply by melvincwng** [reply]
@sjackman, if this reply has helped you, please kindly 'Mark as answer' so that others are aware of the solution, should they encounter the same issue as you.

Thank you for your cooperation, and have a great day ahead 😃

**Reply by mashirozx** [reply]
Doesn't work for me

**Reply by akinolu52** [reply]
Doesn't work for me

**Reply by EricPuskas** [reply]
I had to log out of github as well, and relog, through 2 factor auth.  Followed the same steps as described above and it worked for me afterwards. Good luck

**Reply by Mado13** [reply]
doesn't work for me

**Reply by davecheney** [reply]
@Mado13 I can confirm you have access to copilot via your coupon. 
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic? 
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Comment by dht** [comment]
After following the suggestions and still having the issue, I realized this issue can occur when there is a general issue with co-pilot's servers as seen on [github status](https://www.githubstatus.com/).
